src/components/pages/Theme/Theme.jsx

```diff
@@ -336,6 +346,8 @@ export const Theme = () => {
                   key={`input-type-${index}`}
                   label={type}
                   type={type}
+                  placeholder={`${type} input...`}
```

Comment:
Can we add a `toTitleCase()` utils function that converts these placeholders to title case? Or we could just use the word "Input" for all of them.